tom_tns/tns_api.py

```diff
@@ -97,24 +106,38 @@ def get_reverse_tns_values(option_list, value):
 
 def populate_tns_values():
     """pull all the values from the TNS API and Cache for an hour"""
-
-    # Need to spoof a web based user agent or TNS will block the request :(
-    SPOOF_USER_AGENT = 'Mozilla/5.0 (X11; Linux i686; rv:110.0) Gecko/20100101 Firefox/110.0.'
-
-    # Use sandbox URL if no url found in settings.py
-    tns_base_url = get_tns_credentials().get('tns_base_url', 'https://sandbox.wis-tns.org/api')
     all_tns_values = {}
     reversed_tns_values = {}
-    try:
-        resp = requests.get(urljoin(tns_base_url, 'api/values/'),
-                            headers={'user-agent': SPOOF_USER_AGENT})
-        resp.raise_for_status()
-        all_tns_values = resp.json().get('data', {})
+    if submit_through_hermes():
```

Comment:
This triggers getting values from hermes if `'ENABLE_TNS': True` whether or not the local TNS credentials exist. 
Is this the desired behavior, or should we always use local TNS credentials unless we can't find them, and then use Hermes if `ENABLE_TNS` is set?

Reply:
So the way I thought it would work is setting ENABLE_TNS to True is an intentional thing someone has to do, and is only for submitting to TNS through hermes, so by setting that, they are saying they want to use HERMES to submit to TNS, not use separate TNS credentials. We can't use separately defined TNS credentials to submit through HERMES since HERMES doesn't know about those credentials stored in your TOM. They must be set in your HERMES profile, which associates them with your account and then anything sent via your account will use them.